packages/google-pubsub-microservice/test/GooglePubSubServer.test.ts

```diff
@@ -0,0 +1,100 @@
+import * as delay from 'delay';
+
+import { GCPubSubServer } from '../src';
+import { SUBSCRIPTION_NAME } from './test-app/app.controller';
+import { AppService } from './test-app/app.service';
+import { getTestingApplication } from './test-app/main';
+
+const Emulator = require('google-pubsub-emulator');
+
+describe('GooglePubSubServer', () => {
+  let emulator: any;
```

Comment:
```suggestion
  let emulator: Emulator;
```
?

Reply:
Emulator is not a type, and since it does not have type definitions, I cannot do this

Reply:
Oh I did not see the require, my bad.